lib/shared/permissions.js

```diff
@@ -145,22 +146,13 @@ exports.elevateCommand = (command, options) => {
   const prefixedCommand = _.concat(exports.getEnvironmentCommandPrefix(options.environment), command);
 
   if (os.platform() === 'win32') {
-    const elevator = Bluebird.promisifyAll(require('elevator'));
-
-    return elevator.executeAsync(prefixedCommand, {
-      hidden: true,
-      terminating: true,
-      doNotPushdCurrentDirectory: true,
-      waitForTermination: true
-    }).then(() => {
+    const elevator = Bluebird.promisifyAll(bindings.load('elevator'));
+    return elevator.elevateAsync(_.concat([
+      'cmd.exe',
```

Comment:
Right, that makes sense. The only downside is that the UAC prompt asks to elevate "Command Prompt" which I've always thought looks a bit weird :-/
How hard would it be to pass the "necessary info" as command-line arguments instead of environment variables? (which would mean we wouldn't then need to rely on `cmd.exe`?)